patches/components-variations-variations_seed_store.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/variations/variations_seed_store.cc b/components/variations/variations_seed_store.cc
+index c3567148f8abfb40537d3ca834cb9b5164e3a664..1967f597acb9ceb9a0cd935c5cf0c9ef415fe356 100644
+--- a/components/variations/variations_seed_store.cc
++++ b/components/variations/variations_seed_store.cc
+@@ -57,6 +57,7 @@ constexpr char kIdenticalToSafeSeedSentinel[] = "safe_seed_content";
+ VerifySignatureResult VerifySeedSignature(
+     const std::string& seed_bytes,
+     const std::string& base64_seed_signature) {
++  BRAVE_VERIFY_SEED_SIGNATURE
```

Comment:
We should check with security whether we can go to production without verifying these signatures cc @diracdeltas

Reply:
@moritzhaller any reason not to verify it since you have a signing script for the seed already? (https://github.com/brave/brave-variations/blob/e1953f1c9bdb359f590df11932cc1b313bf16d1a/generate_keys.py)

that said, i'm not sure exactly why a signature would be needed if HTTP fallback is disabled.

Reply:
the reason for the signature is that chromium doesn't consider TLS a sufficient sign of trustworthiness for code that changes browser behavior: https://chromium.googlesource.com/chromium/src/+/master/docs/security/rule-of-2.md#verifying-the-trustworthiness-of-a-source. for parity, we should either verify the ECDSA signatures OR pin the TLS leaf certificate in Brave's key pins store. verifying the signature is better.

Reply:
@diracdeltas sounds good, will add the signature check

Reply:
@diracdeltas the seed file is now being signed with https://github.com/brave/brave-variations/blob/main/crypto.go. The signature will be added to the response header by fastly.

Reply:
also changed the patch from early return to containing our public key